CHANGES.md

```diff
@@ -1,5 +1,30 @@
 # Changes
 
+## edge-29.9.3
+
+This edge release updates the proxy's dependency on the `rustls` library to
+patch security vulnerability [RUSTSEC-2023-0052]  (GHSA-8qv2-5vq6-g2g7), a
+potential CPU usage denial-of-service attack when acceting a TLS handshake from
+an untrusted peer with a maliciously-crafted certificate. Furthermore, this
+edge release contains a few improvements to the control plane and jaeger
+extension Helm charts.
+
+* Addressed security vulnerability [RUSTSEC-2023-0052] in the proxy by updating
+  its dependency on the `rustls` library
+* Added a `prometheusUrl` field for the heartbeat job in the control plane Helm
+  chart (thanks @david972!) ([#11343]; fixes [#11342])
+* Introduced support for arbitrary labels in the `podMonitors` field in the
+  control plane Helm chart (thanks @jseiser!) ([#11222]; fixes [#11175])
+* Added support for config merge and Deployment environment to
+  `opentelemetry-collector` in the jaeger extension (thanks @iAnomaly!)
+  ([#11283])
+
+[#11283]: https://github.com/linkerd/linkerd2/pull/11283
+[#11222]: https://github.com/linkerd/linkerd2/pull/11222
+[#11175]: https://github.com/linkerd/linkerd2/issues/11175
+[#11343]: https://github.com/linkerd/linkerd2/pull/11343
+[#11342]: https://github.com/linkerd/linkerd2/issues/11342
+
```

Comment:
```suggestion
[RUSTSEC-2023-0052]: https://rustsec.org/advisories/RUSTSEC-2023-0052.html

```

Reply:
Afaik it's not going to work since it's already declared in the file and the linter throws errors.